recipes/basemap-data-hires/meta.yaml

```diff
@@ -1,4 +1,4 @@
-package:
+ package:
```

Comment:
This is why I almost always use `git add -p`. That way I can go, "Woops! That shouldn't be there.", just one more time. Sometimes that is just enough times. :smile: